deps/spidershim/src/v8isolate.cc

```diff
@@ -193,6 +193,22 @@ void Isolate::AddStackTrace(StackTrace* trace) {
   pimpl_->stackTraces.push_back(trace);
 }
 
+void Isolate::AddGCPrologueCallback(
+  GCCallback callback, GCType gc_type_filter) {
+  // TODO: used only with dtrace/etw, could be implemented with JS_SetGCCallback
```

Comment:
@brendandahl given that we already have `OnGC()` in `v8isolate.cc`, any reason why we wouldn't just hook these up for reals? :)